ha_ibmdb2i.cc

```diff
@@ -3578,8 +3577,9 @@ void ha_ibmdb2i::setIndexReadEstimate(uint index, ha_rows rows)
 
 ha_rows ha_ibmdb2i::getIndexReadEstimate(uint index)
 {
+  ha_rows i = 1;
   if (indexReadSizeEstimates)
-    return max(indexReadSizeEstimates[index], 1);
+    return std::max(indexReadSizeEstimates[index], i);
```

Comment:
```suggestion
    return std::max(indexReadSizeEstimates[index], ha_rows{1});
```

Don't need the extraneous variable definition, without having the verbose static_cast syntax